test/Microsoft.ML.OnnxTransformerTest/Microsoft.ML.OnnxTransformerTest.csproj

```diff
@@ -27,6 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <NativeAssemblyReference Include="CpuMathNative" />
```

Comment:
I don't know if to include CpuMathNative into the OnnxTransformerTest.csproj the only thing I had to do was to add this line of code (which I did manually). I wonder if I had to do something else.

Reply:
So I discussed this offline with @codemzs and all of these is fine. So I'll mark this as resolved.